tests/test_connections.py

```diff
@@ -0,0 +1,7 @@
+import urllib.request
+import pytest
+
+SITE_LOCATION = "http://localhost:5000"
+
+def test_connection():
+    assert urllib.request.urlopen(SITE_LOCATION) == 200 or 201 or 202
```

Comment:
Did you run this? This syntax probably does not do what you think since the precedence of the `or` operator is lower then the precedence of the `==` operator, and even if it wasn't, the expression `200 or 201 or 202` evaluates to `200`.

Also, looking at the docs for `urlopen` - they mention that:
1. It returns an object and not a number
2. The returned object is a context manager - this means you need to use it with a `with` statement that ensures it is closed when you are done with it